galgebra/printer.py

```diff
@@ -724,12 +788,24 @@ def latex(expr, **settings):
             return s
 
 
-def latex(expr, **settings):
+def latex(expr, **settings) -> str:
+    """
+    Get the latex representation of expr using :class:`GaLatexPrinter`.
+
+    Takes the same options as :func:`sympy.latex`; see that function for
```

Comment:
We should cross link to sympy for this

Reply:
As introduced in #328

Reply:
```suggestion
    Takes the same options as :py:func:`sympy.latex`; see that function for
```

Reply:
Maybe this would work.

Reply:
No, the cause is that I need the fully-qualified name not the alias.